app.py

```diff
@@ -0,0 +1,43 @@
+import config
+from flask import Flask
+from app.api.v1 import use_api
+from celery import Celery
+
+app = Flask(__name__)
+
+def make_celery(app):
+    celery = Celery(
+        app.import_name,
+        backend=app.config['CELERY_RESULT_BACKEND'],
+        broker=app.config['CELERY_BROKER_URL']
+    )
+    celery.conf.update(app.config)
+
+    class ContextTask(celery.Task):
+        def __call__(self, *args, **kwargs):
+            with app.app_context():
+                return self.run(*args, **kwargs)
+
+    celery.Task = ContextTask
+    return celery
+
+app.config.update(
+    CELERY_BROKER_URL='redis://localhost:6379',
+    CELERY_RESULT_BACKEND='redis://localhost:6379'
```

Comment:
Same as with my comment about `config.py`. These variables should be configurable via ENV variables

Reply:
fixed